recipe/meta.yaml

```
@@ -7,12 +7,12 @@ package:
 source:
   fn: reproject-{{version}}.tar.gz
   url: https://pypi.io/packages/source/r/reproject/reproject-{{ version }}.tar.gz
-  md5: e0280e7627eb26dc0dcd62c145a15f18
+  md5: ff2821e83ac2d68a8fecdee1ebcc2d45
 
 build:
   number: 0
-  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
-  skip: True  # [py2k]
+  script: python setup.py install --no-git --single-version-externally-managed --record record.txt
```

This change fixes a build failure for me:

```suggestion
  script: '{{ PYTHON }} -m pip install . -vv'
```